merino-cache/Cargo.toml

```diff
@@ -5,7 +5,7 @@ edition = "2018"
 
 [dependencies]
 anyhow = "^1"
-arc-swap = "1.3"
```

Comment:
Is there something in 1.3.2 that we need?

Reply:
No, this was in part due to the rust version update and making sure we got a good compile. We can probably relax that fairly quickly. If you like, I can add a comment.

Reply:
No, I think it's fine as is. I just wanted to make sure I wasn't missing something in their changelog.